pkg/controller/restic_repository_controller.go

```diff
@@ -120,9 +120,19 @@ func (c *resticRepositoryController) processQueueItem(key string) error {
 	// Don't mutate the shared cache
 	reqCopy := req.DeepCopy()
 
-	switch req.Status.Phase {
-	case "", v1.ResticRepositoryPhaseNew:
+	if req.Status.Phase == "" || req.Status.Phase == v1.ResticRepositoryPhaseNew {
 		return c.initializeRepo(reqCopy, log)
+	}
+
+	// Check both ready and not-ready repos for stale locks, but if this fails
```

Comment:
I would change `Check both ready and not-ready` to be `Check ready or not-ready`. Otherwise it sounds like you are checking multiple repos at a time but it's not the case.